<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="5" passed="2" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-04-21T10:32:46 IST" name="Default suite" finished-at="2022-04-21T10:32:48 IST" duration-ms="2202">
    <groups>
    </groups>
    <test started-at="2022-04-21T10:32:46 IST" name="Default test" finished-at="2022-04-21T10:32:48 IST" duration-ms="2202">
      <class name="POM.TestExecution">
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:POM.TestExecution@6548bb7d]" started-at="2022-04-21T10:32:46 IST" name="beforeTest" finished-at="2022-04-21T10:32:46 IST" duration-ms="19" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:POM.TestExecution@6548bb7d]" started-at="2022-04-21T10:32:46 IST" name="beforeClass" finished-at="2022-04-21T10:32:46 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:POM.TestExecution@6548bb7d]" started-at="2022-04-21T10:32:46 IST" name="beforeMethod" finished-at="2022-04-21T10:32:46 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="test1()[pri:0, instance:POM.TestExecution@6548bb7d]" started-at="2022-04-21T10:32:46 IST" name="test1" finished-at="2022-04-21T10:32:46 IST" duration-ms="13" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:POM.TestExecution@6548bb7d]" started-at="2022-04-21T10:32:46 IST" name="afterMethod" finished-at="2022-04-21T10:32:46 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:POM.TestExecution@6548bb7d]" started-at="2022-04-21T10:32:46 IST" name="beforeMethod" finished-at="2022-04-21T10:32:46 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="testa()[pri:0, instance:POM.TestExecution@6548bb7d]" started-at="2022-04-21T10:32:46 IST" name="testa" finished-at="2022-04-21T10:32:46 IST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testa -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:POM.TestExecution@6548bb7d]" started-at="2022-04-21T10:32:46 IST" name="afterMethod" finished-at="2022-04-21T10:32:46 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:POM.TestExecution@6548bb7d]" started-at="2022-04-21T10:32:46 IST" name="beforeMethod" finished-at="2022-04-21T10:32:46 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="login()[pri:1, instance:POM.TestExecution@6548bb7d]" started-at="2022-04-21T10:32:46 IST" name="login" finished-at="2022-04-21T10:32:48 IST" duration-ms="2024" status="FAIL">
          <exception class="org.testng.internal.thread.ThreadTimeoutException">
            <message>
              <![CDATA[Method POM.TestExecution.login() didn't finish within the time-out 2000]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method POM.TestExecution.login() didn't finish within the time-out 2000
at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:384)
at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:283)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:604)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.thread.ThreadTimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:POM.TestExecution@6548bb7d]" started-at="2022-04-21T10:32:48 IST" name="afterMethod" finished-at="2022-04-21T10:32:48 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method signature="homePage()[pri:2, instance:POM.TestExecution@6548bb7d]" started-at="2022-04-21T10:32:48 IST" name="homePage" finished-at="2022-04-21T10:32:48 IST" depends-on-methods="POM.TestExecution.login" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TestExecution.homePage()[pri:2, instance:POM.TestExecution@6548bb7d] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TestExecution.homePage()[pri:2, instance:POM.TestExecution@6548bb7d] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- homePage -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:POM.TestExecution@6548bb7d]" started-at="2022-04-21T10:32:48 IST" name="afterClass" finished-at="2022-04-21T10:32:48 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:POM.TestExecution@6548bb7d]" started-at="2022-04-21T10:32:48 IST" name="afterTest" finished-at="2022-04-21T10:32:48 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- POM.TestExecution -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
